子子xelz11111222221111122222z1111122222zi1111122222z11111222221111122222子11111
11111
22222

11111
22222
子
11111

11111
zi
11111

11111
子
11111

11111
z
11111
z
11111

11111
子
11111

11111

11111
子
11111
Traceback (most recent call last):
  File "wangwang.py", line 42, in query
    keyword = u'%%%s%%' % keyword.strip()
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 0: ordinal not in range(128)


11111

11111
子
11111
Traceback (most recent call last):
  File "wangwang.py", line 43, in query
    c.execute('''SELECT OID,* FROM contact WHERE nickname_pinyin LIKE ? OR nickname LIKE ? OR user_id LIKE ? LIMIT 10''', (keyword, keyword, keyword))
ProgrammingError: You must not use 8-bit bytestrings unless you use a text_factory that can interpret 8-bit bytestrings (like text_factory = str). It is highly recommended that you instead just switch your application to Unicode strings.



子


子

te
te

子


done


done
子
子
Traceback (most recent call last):
  File "wangwang.py", line 43, in query
    c.execute('''SELECT OID,* FROM contact WHERE nickname_pinyin LIKE ? OR nickname LIKE ? OR user_id LIKE ? LIMIT 10''', (keyword, keyword, keyword))
ProgrammingError: You must not use 8-bit bytestrings unless you use a text_factory that can interpret 8-bit bytestrings (like text_factory = str). It is highly recommended that you instead just switch your application to Unicode strings.

done


done


done


done
子
子
Traceback (most recent call last):
  File "wangwang.py", line 43, in query
    c.execute(u'''SELECT OID,* FROM contact WHERE nickname_pinyin LIKE ? OR nickname LIKE ? OR user_id LIKE ? LIMIT 10''', (keyword, keyword, keyword))
ProgrammingError: You must not use 8-bit bytestrings unless you use a text_factory that can interpret 8-bit bytestrings (like text_factory = str). It is highly recommended that you instead just switch your application to Unicode strings.

done


%%
done


%%
done
子
子
%子%
Traceback (most recent call last):
  File "wangwang.py", line 44, in query
    c.execute(u'''SELECT OID,* FROM contact WHERE nickname_pinyin LIKE ? OR nickname LIKE ? OR user_id LIKE ? LIMIT 10''', (keyword, keyword, keyword))
ProgrammingError: You must not use 8-bit bytestrings unless you use a text_factory that can interpret 8-bit bytestrings (like text_factory = str). It is highly recommended that you instead just switch your application to Unicode strings.

done


%%
done


%%
done
子
子
%子%
Traceback (most recent call last):
  File "wangwang.py", line 44, in query
    c.execute(u'SELECT OID,* FROM contact WHERE nickname_pinyin LIKE ? OR nickname LIKE ? OR user_id LIKE ? LIMIT 10', (keyword, keyword, keyword))
ProgrammingError: You must not use 8-bit bytestrings unless you use a text_factory that can interpret 8-bit bytestrings (like text_factory = str). It is highly recommended that you instead just switch your application to Unicode strings.

done


%%
done


%%
done
子
子
%子%
Traceback (most recent call last):
  File "wangwang.py", line 44, in query
    c.execute(u'SELECT OID,* FROM contact WHERE nickname_pinyin LIKE ? OR nickname LIKE ? OR user_id LIKE ? LIMIT 10', (keyword, keyword, keyword, ))
ProgrammingError: You must not use 8-bit bytestrings unless you use a text_factory that can interpret 8-bit bytestrings (like text_factory = str). It is highly recommended that you instead just switch your application to Unicode strings.

done


%%
done


%%
done
子
子
%子%
done


%%
done


%%
done
子
子
%子%
done

done

done
zi
done
z
done

done
子
done


子

Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    item = alp.Item(title=u'旺旺 \'%s\'' % nickname, subtitle=signature, uid=str(
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 0: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    item = alp.Item(title=u'旺旺 \'%s\'' % nickname, subtitle=signature, uid=str(
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 0: ordinal not in range(128)

子
Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    item = alp.Item(title=u'旺旺 \'%s\'' % nickname, subtitle=signature, uid=str(
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 0: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    title = u'旺旺 \'%s\'' % nickname
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 0: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    title = u'旺旺 \'%s\'' % nickname
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 0: ordinal not in range(128)

子
Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    title = u'旺旺 \'%s\'' % nickname
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 0: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg='aliim:sendmsg?touid=%s' % userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)

zi\ 
zi
Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg='aliim:sendmsg?touid=%s' % userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 8: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg='aliim:sendmsg?touid=%s' % userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)

子
Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg='aliim:sendmsg?touid=%s' % userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 8: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg=u'aliim:sendmsg?touid=%s' % userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg=u'aliim:sendmsg?touid=%s' % userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)

子
Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg=u'aliim:sendmsg?touid=%s' % userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 8: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    title = '旺旺 \'%s\'' % nickname
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 0: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    title = '旺旺 \'%s\'' % nickname
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 0: ordinal not in range(128)

子
Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    title = '旺旺 \'%s\'' % nickname
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 0: ordinal not in range(128)



子

子


子

Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    title = u'旺旺 \'%s\'' % nickname
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 0: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    title = u'旺旺 \'%s\'' % nickname
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 0: ordinal not in range(128)

子
Traceback (most recent call last):
  File "wangwang.py", line 54, in query
    title = u'旺旺 \'%s\'' % nickname
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 0: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg='aliim:sendmsg?touid=%s' % userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg='aliim:sendmsg?touid=%s' % userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)

子
Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg='aliim:sendmsg?touid=%s' % userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 8: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg='aliim:sendmsg?touid=%s' % str(userid), valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg='aliim:sendmsg?touid=%s' % str(userid), valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)

子
Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg='aliim:sendmsg?touid=%s' % str(userid), valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 8: ordinal not in range(128)

xelz
中文

Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg=userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)

zi
Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg=userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 8: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg=userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)

子
Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon='%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg=userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 8: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon=u'%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg=userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)


Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon=u'%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg=userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe9 in position 8: ordinal not in range(128)

子
Traceback (most recent call last):
  File "wangwang.py", line 56, in query
    index+1), icon=u'%s/%s/CustomHeadImages/%s.jpg' % (path, account, userid), arg=userid, valid=True, autocomplete=pinyin)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe5 in position 8: ordinal not in range(128)


zi
z

子
